docs/commands/lint.md

```diff
@@ -1,16 +1,12 @@
 # `lint`
 
-## Introduction
+## Why you need it
 
-Redocly OpenAPI CLI can identify and report on problems found in OpenAPI definitions. This helps you avoid bugs and make API definitions more consistent.
+One of the most powerful features of Redocly OpenAPI CLI is its ability to identify problems found in OpenAPI definition files. This is achieved through the `lint` command, which helps your definitions conform to a consistent set of rules.
 
-The `lint` command reports on problems and executes preprocessors and rules. Unlike the `bundle` command, `lint` doesn't execute decorators.
+The `lint` command ensures that your definition file structure is valid (according to the OpenAPI Specification and your own custom rules, if any) and contains no errors. It also executes preprocessors and rules (unlike the `bundle` command, `lint` doesn't execute decorators). To learn more about preprocessors and rules, see the [custom rules](../resources/custom-rules.md) page.
```

Comment:
suggestion: I would keep the information about not executing decorators more visible - it's an important distinction between those two commands, and if we keep it in parentheses as a sidenote, it's easy to miss.